__tests__/api.tests/general.test.js

```diff
@@ -18,4 +18,112 @@ describe('GET topicccccs', () => {
             expect(body.msg).toBe('Bad api endpoint');
         });
     })
+})
+
+describe('topics', () => {
+    test('GET /api/topics returns with status code 200 and all the topics', () => {
+        return request(app).get('/api/topics').expect(200).then(({body: topics}) => {
+            topics.forEach(topic => {
+                expect(topic).toEqual(expect.objectContaining({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                }))
+            })
+        });
+    })
+})
+
+describe('/api/articles/:article_id/comments', () => {
+    test('200: GET /api/articles/:article_id/comments responds status 200 with an array of comment objects for a specific article, ordered last comment first', () => {
+        return request(app).get('/api/articles/3/comments').expect(200).then(({body: comments}) => {
+            expect(comments.comments).toHaveLength(2);
+            expect(comments.comments).toBeSortedBy('created_at', {descending: true});
+            comments.comments.forEach(comment => {
+                expect(comment).toEqual(expect.objectContaining({
+                    comment_id: expect.any(Number),
+                    body: expect.any(String),
+                    article_id: expect.any(Number),
+                    author: expect.any(String),
+                    votes: expect.any(Number),
+                    created_at: expect.any(String)
+                }))
+            })
+        });
+    })
+    test('200: GET /api/articles/:article_id/comments responds with status code 200 and returns an empty array when there are no comments for that article', () => {
```

Comment:
This is a really good test, and the right way to go about this. Just because an article doesnt have comments, it doesnt always mean theres an error!